ui/src/components/Description/index.tsx

```diff
@@ -26,6 +26,34 @@ const Description: React.FC<Props> = (props) => {
   const { catalog, kind, name } = props;
   const resource = resources.resources.get(`${catalog}/${titleCase(kind)}/${name}`);
 
+  const { webURL, version } = resource.displayVersion;
+
+  const resourceDirUrl = webURL.slice(0, webURL.indexOf(name));
+  const resourceWebUrl = resourceDirUrl + `${name}`;
+
+  // This function transform relative uri of readme into absoulte uri
+  const transformUri = (uri: string) => {
+    if (!uri.includes('./') && !uri.includes('https') && !uri.includes('http')) {
```

Comment:
I think it should check for `http` or `https`

Reply:
I think should check `http` only